bundle/downstream/manifests/quay-bridge-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,513 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    olm.skipRange: ">=3.5.x <3.6.0"
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "quay.redhat.com/v1",
+          "kind": "QuayIntegration",
+          "metadata": {
+            "name": "quay"
+          },
+          "spec": {
+            "clusterID": "openshift",
+            "credentialsSecret": {
+              "name": "quay-credentials",
+              "namespace": "openshift-operators"
+            },
+            "insecureRegistry": false
+          }
+        }
+      ]
+    capabilities: Full Lifecycle
+    categories: OpenShift Optional
+    containerImage: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-bridge-operator-rhel8:v3.6.0
+    createdAt: 2021-07-09 14:07 UTC
+    description: Enhance OCP using Red Hat Quay container registry
+    operators.openshift.io/infrastructure-features: '["Disconnected"]'
```

Comment:
The capitalized version is still supported, but it's supposed to be just all lowercase, "disconnected".